patterns/core/sass/_plugins.scss

```diff
@@ -3,3 +3,5 @@
 //////////////////////////////
 @import '../../../bower_components/sass-toolkit/stylesheets/toolkit';
 @import '../../../bower_components/ibm-colors/ibm-colors';
```

Comment:
should we make a variable for the bower url reference DRY this out? `$bower-url: '../../../bower_components/'`